lib/shipping_easy/configuration.rb

```diff
@@ -13,19 +13,29 @@
 #
 module ShippingEasy
   class Configuration
+    LEGACY_URL = "https://app.shippingeasy.com"
+    DEFAULT_URL = "https://api.shippingeasy.com"
 
+    attr_reader   :base_url
     attr_accessor :api_key,
                   :api_secret,
                   :partner_api_key,
                   :partner_api_secret,
                   :api_version,
-                  :base_url,
                   :http_adapter
 
     # Creates a configuration object, setting the default attributes.
     def initialize
       @http_adapter = ShippingEasy::Http::FaradayAdapter
-      @base_url = "https://app.shippingeasy.com"
+      @base_url = DEFAULT_URL
+    end
+
+    def base_url=(val)
+      if val == LEGACY_URL
+        @base_url = DEFAULT_URL
```

Comment:
Might want to print a warning here that the old URL is deprecated. May save someone from going crazy when they set it to `app.` and then see requests for `api.`.